Makefile

```diff
@@ -130,32 +221,31 @@ CXXFLAGS = \
 	-Wextra \
 	-Werror \
 	-Wshadow \
-	-Wno-c99-designator \
 	-pedantic \
-	-ferror-limit=1 \
 	-std=c++98 \
+	-ferror-limit=1 \
 	-MMD \
 	-MP \
 	-I${PRV_DIR} \
 	-I${INC_DIR}
 
 ifeq (${DEBUG}, 1)
-	CXXFLAGS += -g -DDEBUG=1
+	CXXFLAGS += -gdwarf-4 -DDEBUG=1
```

Comment:
This is because I encountered problems related to the `dwarf` version when compiling with `-g` (`clang` compiles with a version that valgrind doesn't support yet by default). Doing this that way doesn't change anything at use.